src/org/labkey/test/components/ui/grids/TabSelectionExportDialog.java

```diff
@@ -0,0 +1,76 @@
+package org.labkey.test.components.ui.grids;
+
+import org.jetbrains.annotations.Nullable;
+import org.labkey.test.Locator;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.labkey.test.components.html.Checkbox;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.io.File;
+import java.util.Collection;
+
+public class TabSelectionExportDialog extends ModalDialog
+{
+    private static final String TITLE = "Select the Tabs to Export";
+    public TabSelectionExportDialog(WebDriver driver)
+    {
+        super(new ModalDialogFinder(driver).withTitle(TITLE).waitFor().getComponentElement(), driver);
+    }
+
+    public void clickCancel()
+    {
+        elementCache().cancelButton.click();
+    }
+
+    public File export(@Nullable Collection<String> tabs)
+    {
+        //If tabs is null use default tab selections
+        if (tabs != null)
+        {
+            for (WebElement checkbox : elementCache().checkboxes)
+            {
+                String text = checkbox.getText();
+                if (!tabs.contains(text))
+                    elementCache().getCheckBox(checkbox).check();
+                else
+                    elementCache().getCheckBox(checkbox).uncheck();
+            }
```

Comment:
This should throw if a non-existent checkbox is specified.
```suggestion
            Set<String> missingTabs = new HashSet<>(tabs);
            for (WebElement checkbox : elementCache().checkboxes)
            {
                String text = checkbox.getText();
                if (!tabs.contains(text))
                {
                    elementCache().getCheckBox(checkbox).check();
                    missingTabs.remove(text);
                }
                else
                    elementCache().getCheckBox(checkbox).uncheck();
            }
            if (!missingTabs.isEmpty())
            {
                throw new NotFoundException("Tab(s) not found: " + missingTabs);
            }
```